README.building.md

```diff
@@ -1,6 +1,8 @@
 ## Building Freenet from source: Short version
 
-Install junit 4, Apache ant and Hamcrest. For Debian Wheezy these are `junit4`, `ant`, `ant-optional` and `libhamcrest-java`.
+Install junit 4, Apache ant and Hamcrest:
+For Debian Wheezy these are `junit4`, `ant`, `ant-optional` and `libhamcrest-java`.
+For CentOS, "yum install ant-junit" will install all of the above.
```

Comment:
Please use `` `code notation` `` to format commands. If you mention to use `yum`, you should also mention that is should be run as root.